src/cmd/linuxkit/push_openstack.go

```diff
@@ -1,99 +1,37 @@
 package main
 
 import (
-	"bytes"
-	"encoding/json"
 	"flag"
 	"fmt"
-	"io/ioutil"
-	"net/http"
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack"
+	"github.com/gophercloud/gophercloud/openstack/imageservice/v2/imagedata"
+	"github.com/gophercloud/gophercloud/openstack/imageservice/v2/images"
```

Comment:
we usually put all the `github.com` URLs at the bottom (where logrus is). Also you should not need the top two?